pipeline/test_load.py

```diff
@@ -98,3 +98,107 @@ def test_insert_into_earthquake_valid(mock_connection, mock_cursor, valid_earthq
     assert mock_cursor.fetchone.call_count == 8
 
     mock_connection.commit.assert_called_once()
+
+
+def test_insert_into_earthquake_empty_data(mock_connection, mock_cursor):
+    """Test insert_into_earthquake with empty data"""
+    insert_into_earthquake(mock_connection, mock_cursor, [])
+    mock_cursor.fetchone.assert_not_called()
+    mock_connection.commit.assert_not_called()
+
+
+@patch('psycopg2.connect')
+def test_get_connection_success(mock_connect):
+    """Test get_connection with a successful connection."""
+    # Mock the connection object returned by psycopg2.connect
```

Comment:
Can likely remove these comments now, the docstring and variable names etc. all serve as sufficient documentation :)